examples/bond/test/bond-token-flow.js

```diff
@@ -16,28 +16,24 @@ const { buyTxRuntime, issueTx } = require('../scripts/run/common/common');
  * Test for the scenario described in Readme.md
  */
 describe('Bond token Tests', function () {
-  const master = new AccountStore(1000e6);
-  let appManager = new AccountStore(initialBalance);
-  let bondTokenCreator = new AccountStore(initialBalance);
   let issuerAddress = new AccountStore(minBalance);
-  let elon = new AccountStore(initialBalance);
-  let bob = new AccountStore(initialBalance);
-  let dex1 = new AccountStore(initialBalance);
-  let dex2 = new AccountStore(initialBalance);
+  const master = new AccountStore(1000e6);
+  let appManager;
+  let bondTokenCreator;
+  let elon;
+  let bob;
+  let dex1;
+  let dex2;
+
+  const runtime = new Runtime([master, issuerAddress]);
+  [appManager, bondTokenCreator, elon, bob, dex1, dex2] = runtime.defaultAccounts();
 
-  let runtime;
   let flags;
   let applicationId;
   let issuerLsigAddress;
   let lsig;
 
   this.beforeAll(async function () {
```

Comment:
We don't use `await` in this.beforeAll. can we remove async at here?

Reply:
the async here is not needed at all.